.github/workflows/deploy-android-to-firebase-app-distribution.yml

```diff
@@ -0,0 +1,87 @@
+name: deploy-mobile-android-to-firebase-app-distribution
+
+on:
+  workflow_dispatch:
+    inputs:
+      android:
+        description: "Deploy android to firebase app distribution"
+        required: false
+        type: boolean  
+        
+jobs:
+  android:
+    name: android build
+    runs-on: ubuntu-latest
```

Comment:
Could we add the line  `if: inputs.android == true` here just below the the job name. This tells the job to run when the box is checked